core/evancore.vue.libs/src/interfaces.ts

```diff
@@ -83,7 +83,7 @@ export interface EvanFormControlUISpecs {
     size?: number;
     type?: string;
   };
-  type: string;
+  type?: string;
```

Comment:
sure? whats the fallback type, "text"?

Reply:
It's not set in different places. This was a rather quick fix to please the compiler.